README.md

```diff
@@ -66,35 +66,35 @@ end
 
 # now for some tests
 
-When /^I navigate to the google home page$/ do
+When(/^I navigate to the google home page$/) do
```

Comment:
I'm not a big cucumber user, but I personally find the parentheses-less format nicer, more ruby-ish and more readable.
I do however like the use of strings when the regex is not necessary.

Reply:
Fair point. When we migrate to cucumber 3 (at whatever point), the regex style (for us), will become unrequired. As we don't have capture groups, and can use direct strings.